### ISSUE TITLE
Suggestions - cannot arrow up to leave "Search more" option

### ISSUE THREAD

**Issue by amyngu** [OfficeDev/office-ui-fabric-react#3388]
After arrowing down to the option, arrowing up should take the focus to the last suggested item (before the divider)### Bug Report

- __Package version(s)__: office-ui-fabric-react 5.17.0
- __Browser and OS versions__: All

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) Yes, unless SME can

Requested priority: (Blocking, High, Normal, Low) High

Products/sites affected: (if applicable) Any applications using search more with PeoplePicker

#### Describe the issue:

If using search more functionality and arrowing down to the option, you cannot arrow up to get back to the suggestion items above.

#### Actual behavior:
![peoplepicker](https://user-images.githubusercontent.com/31973030/32802878-8af47db0-c936-11e7-99de-b2a9e479f2a9.gif)

After arrowing down to the option, arrowing up takes the focus to the divider and focus cannot be moved back to any preceeding suggestions

#### Expected behavior:

After arrowing down to the option, arrowing up should take the focus to the last suggested item (before the divider)

**Comment by michaelangotti** [comment]
@amyngu Is this still an issue for you? We believe this functionality has been fixed

**Comment by amyngu** [comment]
@michaelangotti Yes this is fixed now!

**Comment by michaelangotti** [comment]
@amyngu Wonderful! I will close this issue then.
